imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -244,7 +266,7 @@ def test_l1a_validate_data_arrays(test_l1a_data: xr.Dataset, index):
     able_to_be_validated = [
         "hi-counters-aggregated",
         "hi-counters-singles",
-        "hi-priority",
+        "hi-omni",
```

Comment:
Do you want to add `hi-omni` in addition to `hi-priority`? Or is this a rename?

Reply:
Oops, thanks for catching this!